### ISSUE TITLE
Is this still being worked on?

### ISSUE THREAD

**Issue by jrwrigh** [CGNS/cgns.github.io#1]
Curious if this is still being worked on. If so, I'd be willing to make some minor (but helpful) syntax changes to docs. Specifically, the mono-spaced text for the node types and function names has been lost when moving from the old HTML site to the new rst-based site. For example:

Old:
![image](https://github.com/CGNS/cgns-modern.github.io/assets/20801821/88d41ce7-ef09-48c7-a050-d8db82a5eeca)
New:
![image](https://github.com/CGNS/cgns-modern.github.io/assets/20801821/fc85cb5a-15db-47e3-a050-e88436f539d5)

**Comment by brtnfld** [comment]
Yes, this is still being worked on. Any help would be appreciated.

**Comment by brtnfld** [comment]
Picking this backup for the 4.5.0 release.

**Comment by MicK7** [comment]
> Curious if this is still being worked on. If so, I'd be willing to make some minor (but helpful) syntax changes to docs. Specifically, the mono-spaced text for the node types and function names has been lost when moving from the old HTML site to the new rst-based site. For example:

You are very welcome to contribute. To get back the monospace text for node types, it was planned to define a custom sphinx "role" and define specific "sids" keywords and ref to handle automatically the syntax coloring/styling and link creation as its done for other language (python, c++, fortran ...). This is still a work in progress. As it is about stylling it not a very high priority.
